packages/neon-api/src/provider/dora/class.ts

```diff
@@ -1,15 +1,24 @@
 import { logging, settings, u, wallet } from "@cityofzion/neon-core";
-import { PastTransaction, Provider } from "../common";
+import { IAddressAbstract, PastTransaction, Provider } from "../common";
 import {
   getBalance,
   getClaims,
   getHeight,
   getMaxClaimAmount,
   getRPCEndpoint,
+  getAddressAbstracts,
 } from "./core";
 
 const log = logging.default("api");
-export class Dora implements Provider {
+
+interface DoraProvider extends Provider {
+  getAddressAbstracts: (
```

Comment:
I would imagine this should be singular as it returns only 1 abstract? So `getAddressAbstract`?

Reply:
the endpoint of web service is "get_address_abstracts" so I used the plural make more sense

Reply:
Ok so it looks like the `Entry` is actually the `AddressAbstract`? And your current `AddressAbstract` is just a container/header for it? If so, please resolve this by renaming `Entry` to `AddressAbstract`.

Very confusing as your raw data interface was named `DoraAddressAbstracts` but your common interface is `AddressAbstract`. Please amend that.